common/views/components/CivicUK/index.tsx

```diff
@@ -106,26 +106,55 @@ const CivicUK = (props: Props) => (
                 label: '<h2 ${headingStyles}>Measure website use</h2>',
                 description:
                   '<ul><li>We use these cookies to recognise you, to count your visits to the website, and to see how you move around it.</li><li>They help us to provide you with a good experience while you browse, for example by helping to make sure you can find what you need.</li><li>They also allows us to improve the way the website works.</li></ul>',
-                cookies: [
-                  '_ga',
```

Comment:
The GA cookies should still appear here as they are set on wellcomecollection.org (they're only 3rd party where the cookie is set from another domain).

Reply:
Ahhh yes I actually knew that but am getting a bit mixed up - thanks will re-add everything that applies to our domain!

Reply:
OK I've added some again (line 87)! But I couldn't find more than these in order to figure out which one were 1st or 3rd party in the big list we have, do we think this is fine for now?

I've also added an auth0 generated cookies in the necessary list, although it never seemed to get deleted.